tests/js/spec/views/organizationIncidents/list/index.spec.jsx

```diff
@@ -7,19 +7,22 @@ import OrganizationIncidentsList from 'app/views/organizationIncidents/list';
 
 describe('OrganizationIncidentsList', function() {
   const {routerContext} = initializeOrg();
+  let mock;
+
+  beforeEach(function() {
+    mock = MockApiClient.addMockResponse({
+      url: '/organizations/org-slug/incidents/',
+      body: [{id: '1', name: 'First incident'}, {id: '2', name: 'Second incident'}],
```

Comment:
Can we add an `Incident` in `tests/js/fixtures`?

Reply:
We have another `Incident` type in there right now... Any ideas for an alternate name for those models?

Reply:
Damn. We should definitely rename the existing ones to not include `Incident` in their names. I don't have any ideas atm. `Service-` Messages? Interruptions? Notices?